src/scss/base/_typography.scss

```diff
@@ -32,6 +36,10 @@ body {
     text-rendering: optimizelegibility;
     -webkit-font-smoothing: antialiased;
     -moz-font-smoothing: antialiased;
+
+    .is-fontsLoading--base & {
```

Comment:
More of an optimisation thing – better to pull this out so that all entries use the same styles:

```
html,
p {
  .is-fontsLoading--base & {
      font-family: $font-family-preload !important;
  }
}
```